### ISSUE TITLE
minZoom, noWrap options being ignored

### ISSUE THREAD

**Issue by subblue** [tangrams/tangram#529]
**TO REPRODUCE THE ISSUE, FOLLOW THESE STEPS:**

The `'minZoom` option when added to either the Leaflet map or Tangram layer is ignored:

```
var map = L.map('map', {minZoom: 8, maxZoom: 14});
var layer = Tangram.leafletLayer({
  scene: 'test.yaml',
  noWrap: true,
  minZoom: 8
});
layer.addTo(map);
```

**RESULT:**

You can mouse wheel zoom all the way out to show multiple copies of the globe.

**EXPECTED RESULT:**

To stop at the minZoom setting and not to repeat the map across the screen.

**ENVIRONMENT:**

All browsers, Leaflet v1.0.3, Tangram v0.12.0

**Comment by subblue** [comment]
Here is a JSFiddle showing the problem in more detail:
https://jsfiddle.net/subblue/kyyg7Lxz/

**Comment by bcamper** [comment]
Thanks - you're correct, silly oversight (checks for max zoom, not min zoom...) :)

**Comment by bcamper** [comment]
min zoom fixed in 7a32e8c2aa508737515348cd25b04fe12dce1d20

**Comment by bcamper** [comment]
`noWrap` fixed in 5582fce87fac8081ea6ef24a87bd6831b6df8f97

**Comment by bcamper** [comment]
Released in [v0.12.1](https://github.com/tangrams/tangram/releases/tag/v0.12.1)
